module/VuFind/config/module.config.php

```diff
@@ -653,6 +653,7 @@
     'Install/PerformSecurityFix', 'Install/ShowSQL',
     'LibGuides/Home', 'LibGuides/Results',
     'LibraryCards/Home', 'LibraryCards/SelectCard',
+    'LibraryCards/ConnectShibbolethCard', 'LibraryCards/ConnectNewShibbolethCard',
```

Comment:
The difference between these actions is not obvious from the names. How about naming ConnectNewShibbolethCard to something like ConnectShibbolethCardLogin? Alternatively use a single action and distinquish between the cases using a query parameter, like `/LibraryCards/ConnectShibbolethCard?redirectToLogin=1`.